packages/storefront-sdk/src/client/index.ts

```diff
@@ -0,0 +1,19 @@
+import { createClient } from '@urql/core';
+import type { Client as UrqlClient } from '@urql/core';
+import type { StorefrontClientParams } from '../index.js';
+
+export class StorefrontClient {
+	private readonly graphqlClient: UrqlClient;
```

Comment:
Wonder if we should consider the native JS private feature? 

When using the node repl or a console log, this is what the output looks like for each version: 

TS Private keyword - the output is also the same as console.log
```sh
> testClient = client({storefrontEndpoint: 'http://localhost:8888'})
Ln {
  graphqlClient: e {
    suspense: false,
    'operations$': [Function (anonymous)],
    reexecuteOperation: [Function: reexecuteOperation],
    createRequestOperation: [Function: createRequestOperation],
    executeRequestOperation: [Function: executeRequestOperation],
    executeQuery: [Function: executeQuery],
    executeSubscription: [Function: executeSubscription],
    executeMutation: [Function: executeMutation],
    query: [Function: query],
    readQuery: [Function: readQuery],
    subscription: [Function: subscription],
    mutation: [Function: mutation],
    subscribeToDebugTarget: [Function (anonymous)]
  }
}
```
js native private field with `#`
```sh
testClient = client({storefrontEndpoint: 'http://localhost:8888'})
Fn {}
```

Also worth noting that the node repl gives me autocomplete on the TS private version, but doesn't on the native one.

Reply:
The way that I've been thinking about private fields is not so much "make it so that this field can't be overwritten" but more "make it so that this field doesn't appear in Intellisense." But if you feel there's value in making it [hard private](https://www.typescriptlang.org/docs/handbook/2/classes.html#caveats), I'm happy to make that change.

Reply:
Implemented in https://github.com/getnacelle/nacelle-js/pull/306/commits/7770cd4370b5d6874fb99da5932bc1b0bf2a81ed